src/parser.jl

```diff
@@ -3494,7 +3498,11 @@ function parse_atom(ps::ParseState, check_identifiers=true)
         # +     ==>  +
         # .+    ==>  (. +)
         # .=    ==>  (. =)
-        bump_dotsplit(ps, emit_dot_node=true)
+        if is_dotted(peek_token(ps))
+            bump_dotsplit(ps, emit_dot_node=true)
+        else
+            bump(ps, remap_kind=K"Identifier")
+        end
```

Comment:
`bump_dotsplit` has a remap_kind argument

```suggestion
        bump_dotsplit(ps, emit_dot_node=true, remap_kind=K"Identifier")
```

Reply:
Doesn't work, because that also remaps the dotted version, which is undesirable.

Reply:
I think it works for `SyntaxNode` but then there's an error in `Expr` conversion. So we can leave that for now, I'll open a separate issue for it.  But we should be consistent.

Why did you need to have `bump(ps, remap_kind=K"Identifier")` here anyway?

Reply:
> Why did you need to have `bump(ps, remap_kind=K"Identifier")` here anyway?

Just to avoid having an explit is_operator check on the `sig_kind in KSet"Identifier var $"`. It also seems odd that e.g. `isa` gets turned into Identifiers, but `+` does not. I'm happy with either way though.